pkg/restapi/issuer/operation/operations.go

```diff
@@ -366,6 +371,40 @@ func (c *Operation) registerHandler() {
 	}
 }
 
+func (c *Operation) determineProfile(query url.Values) (*Profile, error) {
```

Comment:
Typically, lower-level details belong lower in the file (and after where they get called from)